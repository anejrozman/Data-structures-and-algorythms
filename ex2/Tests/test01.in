8,32,0
61.71,50.81,75.71000000000001,69.81
0,79.06,12,89.06
36.15,85.97,39.15,88.97
67.94,88.47,89.94,92.47
77.68,13.12,84.68,40.12
34.93,8.2,37.93,30.2
0,60.3,21,64.3
27.79,39.37,33.79,42.37

61.71,50.81
61.71,69.81
75.71000000000001,50.81
75.71000000000001,69.81
0,79.06
0,89.06
12,79.06
12,89.06
36.15,85.97
36.15,88.97
39.15,85.97
39.15,88.97
67.94,88.47
67.94,92.47
89.94,88.47
89.94,92.47
77.68,13.12
77.68,40.12
84.68,13.12
84.68,40.12
34.93,8.2
34.93,30.2
37.93,8.2
37.93,30.2
0,60.3
0,64.3
21,60.3
21,64.3
27.79,39.37
27.79,42.37
33.79,39.37
33.79,42.37
