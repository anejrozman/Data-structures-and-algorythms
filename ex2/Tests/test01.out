148.10929299521422
142.31641867593646
146.7277613648473
145.92331229770446
195.9776185154676
206.79370578877342
175.59689529064806
182.87448080719483
158.7077484875722
160.82946687615933
156.91109882753923
159.306213806797
148.62290499223474
147.8059941221552
142.31641867593646
143.58064346358256
169.6847716329321
153.69072680313636
173.9107418492973
158.59484677892326
150.68678062147356
143.13007533164017
150.61343992460533
144.51668826756466
171.07508152693998
195.3330903378794
153.6271762459075
155.5285815571134
143.96092704284465
143.44139031075377
142.32891760459552
143.1706149364189
